### PR TITLE
reimplements graphene grinding, adds a production method

### PR DESCRIPTION
**Pull request by Mazianni**
🆑 
Reintroduces graphene. Graphene is made by cracking Co2 in an oxygen regenerator. Graphene can be ground into carbon and acetone.
By the request of @ Mama
🆑

### CODE REVIEW

**Review by PsyCommando — CHANGES_REQUESTED** [review]
You really should use graphite for that?

File: code/modules/materials/definitions/materials_mineral.dm [hunk 1]
```diff
@@ -43,6 +43,20 @@
 	ore_matter = list(MATERIAL_GRAPHITE = 2000)
 	energy_combustion = 32.7
 
+/material/graphene
+	name = MATERIAL_GRAPHENE //Aka carbon
```
Comment (PsyCommando) [inline code/modules/materials/definitions/materials_mineral.dm:47]: Graphite is raw carbon, and its made out of graphene layers.

**Comment by Mazianni** [comment]
I don't think graphite's a good option, as literally everybody can get it.

This provides a production path that's one-step, through engineering companies or even government atmos.

**Review by PsyCommando — COMMENTED** [review]
> 
> 
> I don't think graphite's a good option, as literally everybody can get it.
> 
> This provides a production path that's one-step, through engineering companies or even government atmos.

Well, how are people supposed to get it then? Acetone was meant to be easily obtainable as well as anything else, but we forgot about it.

**Comment by Mazianni** [comment]
Graphene is made from Co2 crackers. They are machines, constructable, even, which turn Co2 into hot oxygen and graphene.

**Comment by PsyCommando** [comment]
> 
> 
> Graphene is made from Co2 crackers. They are machines, constructable, even, which turn Co2 into hot oxygen and graphene.

Yeah, but then you can basically make it for free from waste gases you can get anywhere. Instead of having to get it mined.

**Comment by Mazianni** [comment]
I've already implemented another chemical path that allows for acetone production, which already uses graphite. Grind graphite, combine with O2/H2, get acetone. 

The gas generator PR is related so people can get O2/H2 in reagent forms.

I understand why you'd want it to be graphite, but I really think we should be encouraging interaction, and this is one way - atmos is government controlled, so the gov can potentially produce acetone and sell it, or use it in a hospital.

**Pull request merged**